src/telemetry/telemetry.ts

```diff
@@ -65,6 +66,27 @@ class Telemetry implements ITelemetry {
     validateExternalLink(url, componentName);
   }
 
+  public async trackApiCallEvent(componentName: string, url: string,
+    options: IRequestOptions): Promise<Response> {
+    const properties: { [key: string]: string } = {
+      ComponentName: componentName,
+      IsRequestSuccessful: 'false',
+      Url: url
```

Comment:
shouldn't this be sanitized, as it contains query parameters that could have sensitive info.

Reply:
Yes, I think the url will need to be sanitized. I'll check to see if there more that I need to do other than stripping out query parameter values?